lib/bots/versionbot.ts

```diff
@@ -650,17 +650,21 @@ export class VersionBot extends ProcBot {
                 return false;
             });
         }).then((exists: boolean) => {
-            let versionistCommand = 'versionist';
-            if (exists) {
-                versionistCommand = `${versionistCommand} -c versionist.conf.js`;
-                this.logger.log(LogLevel.INFO, 'Found an overriding versionist config ' +
-                    `for ${versionData.repoFullName}, using that`);
-            }
-
-            return Promise.mapSeries([
-                versionistCommand,
-                'git status -s'
-            ], cliCommand);
+            let versionistCommand: string;
+            return this.getNodeBinPath().then((nodePath: string) => {
+                versionistCommand = `${nodePath}${path.sep}versionist`;
+                if (exists) {
+                    versionistCommand = `${versionistCommand} -c versionist.conf.js`;
+                    this.logger.log(LogLevel.INFO, 'Found an overriding versionist config ' +
+                        `for ${versionData.repoFullName}, using that`);
+                }
+                console.log(versionistCommand);
```

Comment:
Leftover console.log (I think? if we do want this, it should at least be `this.logger.log` instead)